loadtest/main.go

```diff
@@ -301,23 +107,36 @@ func generateMessage(config Config, key cryptotypes.PrivKey, batchSize uint64) (
 			Funds:        amount,
 		}
 	case Staking:
-		delegatorAddr := sdk.AccAddress(key.PubKey().Address()).String()
-		chosenValidator := Validators[rand.Intn(len(Validators))].OperatorAddress
-		// Randomly pick someone to redelegate / unbond from
-		srcAddr := ""
-		for k := range DelegationMap[delegatorAddr] {
-			if k == chosenValidator {
-				continue
+		msgType := config.MsgTypeDistr.SampleStakingMsgs()
```

Comment:
are these changes intentional? Looks like we're throwing out some of the logic that's required to ensure we have existing delegations

Reply:
Ah yeah this needs some refactoring. I had decided to use erics version of the staking msg sampling because it allowed for selecting what types of staking message distributions to use but I think the left side one is better, I can refactor